### DISCUSSION TITLE
Can we show Azure Kubernetes Service cost as a dedicated service section

### DISCUSSION THREAD

**Discussion by ajmal-yazdani** [mivano/azure-cost-cli#14] [Ideas]
Can we show cost for an AKS cluster and all resources around AKS like VMSS, Disk, Load Balancer, etc as a dedicated section please same like Azure App Service, etc?

![image](https://user-images.githubusercontent.com/1866370/232490972-78caec40-bbf8-45af-93a3-cb4596e6b794.png)

**Comment by mivano** [comment]
It is not a dedicated service name, so unfortunately, Azure/Microsoft stuffs that all under this `Azure App Service` name. Only way I can think of is to add some sort of filter and filter on resource type or resource group instead.
